src/test/resources/org/openstreetmap/atlas/checks/validation/intersections/MEX_7-24-55-subatlas.txt

```diff
@@ -0,0 +1,2459 @@
+# Nodes
```

Comment:
Where's this file used?

Reply:
https://github.com/osmlab/atlas-checks/pull/217/files#diff-3e0a2f4f82c8d8e8c1d457faa5961d42

Reply:
BigNodeBadDataCheckTestRule